pkg/client/interface_test.go

```diff
@@ -33,18 +33,13 @@ import (
 	fakedynamic "k8s.io/client-go/dynamic/fake"
 )
 
-func newUnstructured(backendVersion, kind, name string) *unstructured.Unstructured {
-	return &unstructured.Unstructured{
-		Object: map[string]interface{}{
-			"apiVersion": fmt.Sprintf("%s/%s", directcsi.Group, backendVersion),
-			"kind":       kind,
-			"metadata": map[string]interface{}{
-				"namespace": metav1.NamespaceNone,
-				"name":      name,
-			},
-		},
-	}
-}
+const (
+	mb20           = 20 * 1024 * 1024
+	testNodeName   = "test-node-1"
+	testDriveName  = "test-drive-1"
+	testTenantName = "tenant-1"
+	tenantLabel    = "direct.csi.min.io/tenant"
```

Comment:
we don't need this label, We don't support filtering on such labels.

For volumes, we support the following labels in filtering,

```
direct.csi.min.io/drive-path
direct.csi.min.io/node
direct.csi.min.io/pod.name
direct.csi.min.io/pod.namespace
```